zkevm-circuits/src/evm_circuit/execution/create.rs

```diff
@@ -789,6 +859,23 @@ mod test {
         } else {
             OpcodeId::CREATE
         });
+
+        // construct address collision by create2 twice
```

Comment:
These codes useless？

Reply:
removed in [52be462](https://github.com/scroll-tech/zkevm-circuits/pull/406/commits/52be46292621459896972f3cac9d66df191894a9) thanks!